cmd/clusterctl/cmd/rollout/resume.go

```diff
@@ -0,0 +1,85 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package rollout
+
+import (
+	"github.com/spf13/cobra"
+	"k8s.io/kubectl/pkg/util/templates"
+	"sigs.k8s.io/cluster-api/cmd/clusterctl/client"
+)
+
+// resumeOptions is the start of the data required to perform the operation.
+type resumeOptions struct {
+	kubeconfig        string
+	kubeconfigContext string
+	resources         []string
+	namespace         string
+}
+
+var resumeOpt = &resumeOptions{}
+
+var (
+	resumeLong = templates.LongDesc(`
+		Resume of cluser-api paused resources.
+
+	        Paused resources will not be reconciled by a controller. By resuming a
+		resource, we allow it to be reconciled again.`)
```

Comment:
Again following the pattern of `kubectl rollout resume -h`.
```suggestion
		Resume a paused cluster-api resource

                 Paused resources will not be reconciled by a controller. By resuming a resource, we allow it to be reconciled again.
Currently only MachineDeployments support being resumed.`)
```

Also I noticed the indentation was off when I printed out the help for this comamand.
```
 $ clusterctl alpha rollout resume -h
Resume of cluser-api paused resources.

     Paused resources will not be reconciled by a controller. By resuming a

 resource, we allow it to be reconciled again.
```

Reply:
Done.